engine/graphics/src/opengl/graphics_opengl.cpp

```diff
@@ -1807,8 +1807,8 @@ static void LogFrameBufferError(GLenum status)
             glGetShaderiv(shader_id, GL_INFO_LOG_LENGTH, &logLength);
             if (logLength > 0)
             {
-                GLchar *log = (GLchar *)malloc(logLength);
-                glGetShaderInfoLog(shader_id, logLength, &logLength, log);
+                log_str = (GLchar *)malloc(logLength);
+                glGetShaderInfoLog(shader_id, logLength, &logLength, log_str);
```

Comment:
This is the main fix. `log_str` were created to read errors there, but never actually used.